src/poset/event.go

```diff
@@ -287,23 +277,20 @@ func (e *Event) Hash() ([]byte, error) {
 }
 
 func (e *Event) Hex() string {
-	if e.Message.Hex == "" {
-		hash, _ := e.Hash()
-		e.Message.Hex = fmt.Sprintf("0x%X", hash)
-	}
-	return e.Message.Hex
+	hash, _ := e.Hash()
+	return fmt.Sprintf("0x%X", hash)
```

Comment:
This change is causing performance degradation as it call fmt.Sprintf() on every call of Hex(), while the current version call this function only on the first call.

Reply:
This prevents duplication of data. 
Which option do you suggest?
I am opposed to storing hex and hash for the same event in memory.
I think that at this stage we need to make the application as easy as possible.
Performance optimization can be done after the application logic is developed.

Reply:
Why was this change made in the first placer?  what problem does it fix?